terraform/dns.tf

```diff
@@ -3,3 +3,7 @@ resource "aws_route53_zone" "private-mgmt" {
   // this argument makes it a Private Hosted Zone
   vpc_id = "${module.mgmt-vpc.vpc_id}"
 }
+
+resource "aws_route53_zone" "public-app" {
+  name = "${var.public_app_zone_name}"
+}
```

Comment:
Hmm, not sure this is something we will be able to manage in Route 53 ourselves, especially if the VPC is behind the firewall. Do we have a card around that? Anyway, I'd suggest leaving this out, for now. @eyard Thoughts?

Reply:
Hard to comment as I don't really have the big picture here.  Is this for use in BSP or for a completely new platform?

Reply:
Talked it over with @LinuxBozo and we decided to remove this resource for now.